File: capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/ScreenApiExampleApp.java
package net.gini.android.capture.screen;

import net.gini.android.capture.example.shared.BaseExampleApp;

/**
 * Created by Alpar Szotyori on 01.12.2017.
 *
 * Copyright (c) 2017 Gini GmbH.
 */

public class ScreenApiExampleApp extends BaseExampleApp {

    @Override
    protected String getClientId() {
        return getString(R.string.gini_api_client_id);
    }

    @Override
    protected String getClientSecret() {
        return getString(R.string.gini_api_client_secret);
    }
}

File: capture-sdk/screen-api-example-app/src/main/java/net/gini/android/capture/screen/ScreenApiExampleApp.kt
package net.gini.android.capture.screen

import androidx.multidex.MultiDexApplication
import dagger.hilt.android.HiltAndroidApp
import net.gini.android.capture.network.GiniCaptureDefaultNetworkService
import javax.inject.Inject


/**
 * Main Entry of the application.
 */

@HiltAndroidApp
class ScreenApiExampleApp : MultiDexApplication() {
    @set:Inject var giniCaptureDefaultNetworkService: GiniCaptureDefaultNetworkService? = null

    fun clearGiniCaptureNetworkInstances() {
        giniCaptureDefaultNetworkService?.cleanup()
        giniCaptureDefaultNetworkService = null
    }

}